docs/01-getting-started/03-react-essentials.mdx

```diff
@@ -334,7 +334,7 @@ With this approach, the rendering of `<ExampleClientComponent>` and `<ExampleSer
 >
 > - This pattern is **already applied** in [layouts and pages](/docs/app/building-your-application/routing/pages-and-layouts) with the `children` prop so you don't have to create an additional wrapper component.
 > - Passing React components (JSX) to other components is not a new concept and has always been part of the React composition model.
-> - This composition strategy works across Server and Client Components because the component that receives the prop has no knowledge of **what** the prop is. It is only responsible for where the thing that it is passed should be placed.
+> - This composition strategy works across Server and Client Components because the component that receives the prop has no knowledge of **what** the prop is. It is only responsible for where the thing that is passed should be placed.
```

Comment:
Can we avoid the ambiguity of `it`? How about this instead:

```suggestion
> - This composition strategy works across Server and Client Components because the component that receives the prop can choose where to place the prop, but has no knowledge of **what** the prop is.
```